checker/tests/signedness/SignednessGlbIssue.java

```diff
@@ -17,7 +17,7 @@ void m2() {
         @Unsigned int c = field;
     }
 
-    void m3() {
-        @Unsigned int c = this.field;
+    void m3(@IntRange(from = 0, to = Integer.MAX_VALUE) int array[]) {
```

Comment:
Why did you replace this rather than adding a test case?  Is the old test wrong?  If it's not wrong, it is usually better to augment rather than delete it.